src/app/content/highlights/selectors.ts

```diff
@@ -95,6 +96,14 @@ export const highlightLocationFilters = createSelector(
     : new Map() as HighlightLocationFilters
 );
 
+export const orderedSummaryHighlights = createSelector(
+  localState,
+  highlightLocationFilters,
+  (state, locationFilters) => {
+    return getSortedSummeryHighlights(state.summary.highlights, locationFilters);
```

Comment:
if what you want is `highlights` you should depend on `summaryHighlights` not `localState` so the memoization works better